pkg/report/linux.go

```diff
@@ -2457,4 +2457,5 @@ var linuxOopses = append([]*oops{
 		[]*regexp.Regexp{},
 		crash.UnknownType,
 	},
+	&gatherGoRuntimeBugs,
```

Comment:
Please add a few tests to `pkg/report/testdata/linux/report`.

Reply:
The three test cases are explained in the commit description, but let me know if there are other cases you had in mind :)